px-vis-toolbar.html

```diff
@@ -398,13 +403,30 @@
           mouseup: 'stopLasso'
         },
         subConfig: {
+          filterIn: {
+           tooltipLabel: 'Filter in',
+           icon: 'px-vis:draw-stripe',
+           disabled: true,
+           onClick: function() {
+            this.fire('apply-lasso-filter', 'filterIn');
+           }
+          },
+          filterOut: {
+           tooltipLabel: 'Filter out',
+           icon: 'px-vis:remove-stripe',
+           disabled: true,
+           onClick: function() {
+             this.fire('apply-lasso-filter', 'filterOut');
+           }
+          },
           clearCrosshair: {
             tooltipLabel: 'Clear lasso data',
             icon: 'px-nav:close',
             onClick: function() {
               this.set('crosshairData', { 'rawData': [], 'timeStamps': []});
               this.set('interactionSpaceConfig.resetLassoData', !this.interactionSpaceConfig.resetLassoData);
               this.set('resetLassoData', !this.resetLassoData);
+              this.fire('diable-lasso-filter');
```

Comment:
Looks like typo in event name, should be 'disable-lasso-filter'.

Reply:
Fixed it